stars/test/test_fleet.py

```diff
@@ -665,24 +665,25 @@ def test_unload3(self):
 
     def test_unload4(self):
         p_ref = reference.Reference(player.Player())
-        f = fleet.Fleet(player=p_ref) + ship.Ship(cargo=cargo.Cargo(titanium=2, lithium=2, silicon=2), cargo_max=10)
-        f2 = fleet.Fleet(player=p_ref) + ship.Ship(cargo_max=10)
-        f.location = location.Location(reference=f2)
-        f.order.ti = 10
-        f.order.li = 100
-        f.order.si = 0
+        f2 = fleet.Fleet(ID='f2', player=p_ref, location=location.Location(0, 0, 1)) + ship.Ship(cargo=cargo.Cargo(), cargo_max=10)
+        f = fleet.Fleet(ID='f', player=p_ref, location=location.Location(0, 0, 1), orders=[order.Order(location=location.Location(reference=f2))]) + ship.Ship(cargo=cargo.Cargo(titanium=2, lithium=2, silicon=2), cargo_max=10)
+        f.orders[0].load_ti = 10
+        f.orders[0].load_li = 100
+        f.orders[0].load_si = 0
```

Comment:
Revert cargo handling
```suggestion
        f.order.ti = 10
        f.order.li = 100
        f.order.si = 0
```